parm/stage/fv3_nest.yaml.j2

```diff
@@ -0,0 +1,17 @@
+{% set cycle_HH = current_cycle | strftime("%H") %}
+{% set cycle_YMD = current_cycle | to_YMD %}
+{% set cycle_YMDH = current_cycle | to_YMDH %}
+{% set ntile = 7 %}
+fv3_nest:
+    mkdir:
+        - "{{ COM_ATMOS_RESTART_PREV }}"
+    copy:
+        {% if EXP_WARM_START %}
```

Comment:
If you want to allow for missing variables, then I would suggest logic to be explicit.  e.g.
```
        {% if EXP_WARM_START == True %}
```

Reply:
Good suggestion, thanks @DavidHuber-NOAA ! Done @ 3f08f035